zad5/db/migrate/20181129135415_add_tracks_to_artist.rb

```diff
@@ -0,0 +1,5 @@
+class AddTracksToArtist < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :artists, :track, foreign_key: true
```

Comment:
to w sumie niepotrzebne przy takiej referencji wychodziłoby, że artysta ma jeden utwór